src/Headers/Simulation.h

```diff
@@ -261,7 +266,7 @@ class Simulation : public SimulationBase
   virtual void ProcessNbodyParameters(void);
   virtual void ProcessParameters(void)=0;
   virtual void RecordDiagnostics(void);
-  virtual void RegulariseParticleDistribution(const int) {};
+  virtual void RegulariseInitialConditions(const int) {};
```

Comment:
Yes, it should be part of both of them (actually the idea was in part from the Gaburov & Nitadori meshless paper so definitely belongs in there).  Once we've figured out the few remaining issues with the regularisation, then we should be able to transfer it over quite easily.